### DISCUSSION TITLE
2022-09-16: ERR#0015: error 403, try again later.

### DISCUSSION THREAD

**Discussion by freezedman** [alvarobartt/investpy#601] [General]
Hello man, the feeling that investing has closed the window :(

**Comment by alvarobartt** [comment]
Hi @freezedman, I'm actively working on this issue, it has already been fixed for the stocks, check its progress at https://github.com/alvarobartt/investpy/issues/600 and also at https://twitter.com/alvarobartt/status/1570661023262310402

**Reply by wssranjula** [reply]
I am getting the same error. :(

**Reply by sasi-mohan** [reply]
I'm getting the same error.
ConnectionError: ERR#0015: error 403

Any idea when this can be fixed?

**Reply by marcnshapiro** [reply]
Also getting this error.  For at least the last 48 - 50 hours.

**Reply by julito1469** [reply]
I have the same problem

**Reply by adelRosal** [reply]
Hey @alvarobartt  is there any way we can help you?

**Reply by adelRosal** [reply]
Support investpy https://github.com/alvarobartt/investpy/discussions/609

**Reply by alvarobartt** [reply]
So I've already contacted Investing.com and started a conversation there, I'll let you all know the outcome of it. Thanks for your patience! :hugs:

**Reply by marcialgarza** [reply]
Thanks Alvaro.  Looking forward to a quick resolution!  ;)

**Reply by gianandreabertello** [reply]
investpy.search_quotes receives the Connection error so retrieve_historical_data cannot work

**Reply by alvarobartt** [reply]
Hi @gianandreabertello just FYI I'm still working on the issue, waiting for Investing.com to get back to me with an answer! Sorry for the inconvenience and thanks for the patience...

**Comment by marcnshapiro** [comment]
Any further news from Investing.com?

Marc

**Reply by alvarobartt** [reply]
Hi @marcnshapiro, no news from Investing.com yet, but I've released [`investiny`](https://github.com/alvarobartt/investiny) in the meantime so as to cover most of the Investing.com functionality but using a different API, as the default one at `api.investing.com` is Cloudflare protected... So I'd recommend all of you to use `investiny` in the meantime, and follow me on both GitHub and Twitter in order to receive updates on `investpy`. Thanks! 🤗
